docs/developer/restoring-database.md

```diff
@@ -0,0 +1,9 @@
+# Restoring a Cloud.gov Database
+
+Documentation for restoring from a backup can be found [here](https://gist.github.com/markdboyd/8f20f9bdcfb591febaa838a50d0deb23)
```

Comment:
Oh I meant copy their content, not the link, should have had you share screen. That's my bad. I'm running through these now so I can make a new branch. I see this one is targetting the ms sandbox. I think we forgot to mention, if you have a PR that is just doc changes, you don't want to start the branch with ms/ as it will deploy to your sandbox which isn't needed for a doc change. (Plus the deploy overwrites any other PR you have up).